daemon/src/model/cfd.rs

```diff
@@ -1234,20 +1234,31 @@ pub fn calculate_long_liquidation_price(leverage: Leverage, price: Price) -> Pri
     price * leverage / (leverage + 1)
 }
 
-/// Returns the Profit/Loss (P/L) as Bitcoin. Losses are capped by the provided margin
 pub fn calculate_profit(
+    payout: SignedAmount,
+    margin: SignedAmount,
+) -> Result<(SignedAmount, Percent)> {
+    let profit = payout - margin;
+    let percent = Decimal::from_f64(100. * profit.as_sat() as f64 / margin.as_sat() as f64)
+        .context("Unable to compute percent")?;
+
+    Ok((profit, Percent(percent)))
+}
+
+/// Returns the Profit/Loss (P/L) as Bitcoin. Losses are capped by the provided margin
+pub fn calculate_profit_from_current_price(
     initial_price: Price,
-    closing_price: Price,
+    current_price: Price,
     quantity: Usd,
     leverage: Leverage,
     position: Position,
-) -> Result<(SignedAmount, Percent)> {
+) -> Result<(SignedAmount, Percent, SignedAmount)> {
     let inv_initial_price =
         InversePrice::new(initial_price).context("cannot invert invalid price")?;
     let inv_closing_price =
-        InversePrice::new(closing_price).context("cannot invert invalid price")?;
+        InversePrice::new(current_price).context("cannot invert invalid price")?;
```

Comment:
Nit: Variable names not aligned.

Reply:
Not sure what you mean by this comment.

Reply:
Line 1258 uses `inv_closing_price` and line 1259 uses `current_price`.